tree_utils.go

```diff
@@ -0,0 +1,31 @@
+package i3
+
+import "strings"
+
+func (n *Node) FindFocusedLeaf() *Node {
```

Comment:
I think it would be easier to understand if we introduced only the `func(n *Node) bool { return n.Focused }` predicate. Can you do that and remove the `FindFocusedLeaf` function?

Reply:
you mean something like this?:
```go
// implementation
package i3
func IsFocused(n *Node)  bool { 
    return n.Focused 
}

....

// usage
package main
var n Node
...
n = GetBlahBlahNode()
n.FindFocused(i3.IsFocused())
```

Reply:
Yes, though now that I see it written out, it would merely be an accessor for the Focused field.

Maybe it would be better to not introduce this one after all. It’s just one line to write out, and fairly straight-forward.

Can you remove please?

Reply:
Done